File: Easy/Python/1684. Count the Number of Consistent Strings.py
class Solution(object):
    def countConsistentStrings(self, allowed, words):
        count = 0

        for s in words:
            # Check if each character in String s is present in allowed
            for i in range(len(s)):
                # If it isn't, exit the loop
                # If each letter is present in allowed, increase count by 1
                if s[i] not in allowed:
                    break
                if i == len(s) - 1:
                    count += 1

        return count

File: Easy/Java/1684. Count the Number of Consistent Strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int count = 0;

        for(String s : words){
            // Check if each character in String s is present in allowed
            for(int i = 0; i < s.length(); i++){
                // If it isn't, exit the loop
                // If each letter is present in allowed, increase count by 1
                if(allowed.indexOf(s.charAt(i)) == -1)
                    break;
                if(i == s.length() - 1)
                    count++;
            }
        }

        return count;
    }
}
